components/publishers/index.jsx

```diff
@@ -0,0 +1,167 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+
+import ClassNames from 'classnames';
+
+import { PUBLISHERS } from '../../utilities/constants';
+
+import Avatar from '../avatar';
+import MediaObject from '../media-object';
+import Button from '../button';
+
+// This component's `checkProps` which issues warnings to developers about properties when in development mode (similar to React's built in development tools)
+import checkProps from './check-props';
+import componentDoc from './docs.json';
+
+const propTypes = {
+	/**
+	 * Unique ID for the component to enable ARIA functioning and keyboard interactions
+	 */
+	id: PropTypes.string,
+	/**
```

Comment:
I would recommend an `avatar` prop for use by the comment variant. This would allow reuse and customization of the Avatar component to the consuming developers needs including clicking the Avatar

Reply:
I will add an `avatar` prop and update the PR soon.